debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/IncrementalSnapshotIT.java

```diff
@@ -361,9 +366,7 @@ public void snapshotPartitionedTable() throws Exception {
         startConnector(x -> x.with(PostgresConnectorConfig.TABLE_INCLUDE_LIST, "s1.part, s1.part1, s1.part2"));
         waitForConnectorToStart();
 
-        sendAdHocSnapshotSignal("s1.part");
-        sendAdHocSnapshotSignal("s1.part1");
-        sendAdHocSnapshotSignal("s1.part2");
+        sendAdHocSnapshotSignal("s1.part", "s1.part1", "s1.part2");
```

Comment:
Any specific reason to change this?

Reply:
Reverted change. This was originally required because declaring id column of signal table as primary key was causing pkey constraint violations as each sendAdHocSnapshotSignal was using the same id ('ad-hoc').

With the new signal table structure, with a new sno column (type serial) as primary key, it works fine.